pkg/eval/platform/platform.go

```diff
@@ -35,9 +37,46 @@ import (
 // Whether or not the platform is Microsoft Windows.
 // This is read-only.
 
+//elvdoc:fn hostname &omit-domain=false
+//
+// The host name of the system. If option `&omit-domain` is used any domain
+// components are stripped from the host name. If an error occurs while
+// fetching the host name `$nil` is substituted. This is implemented using
+// Go's [`os.Hostname()`](https://golang.org/pkg/os/#Hostname) function.
```

Comment:
In Go suffixing function names with `()` isn't really a thing; just say `os.Hostname`.

Reply:
Done.